src/components/Footer/styles.js

```diff
@@ -0,0 +1,128 @@
+import styled from '@emotion/styled'
+import { theme } from '../../utils/styles'
+
+export const FooterContainer = styled.div`
+  text-align: center;
+  position: absolute;
+  width: 100%;
+  bottom: 0;
+
+  @media (min-width: ${theme.breakpoints.s}) {
+    flex-direction: row;
+    min-height: 250px;
```

Comment:
Let's avoid settings fixed height here. The content instead should set the height. If it's not exactly like you need it to be padding or margin